### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bitrisekristof**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,6 +200,9 @@ Once installed:
     add the Terminal and the Script Runner apps to `System Preferences - Security and Privacy - Accessability`
     also run the machine-setup/playbook.yml on the machine to enable access to system events
     you will need to run this from your dev machine, because the setting needs manual interaction, so the automation would time out.
+    
+* Add SSH key to authorized keys
+* Add `vagrant` user ad password less sudoer
```
Comment (fehersanyi-bitrise) [inline README.md:205]:
```suggestion
* Add `vagrant` user ad passwordless sudoer
```
passwordless is actually one word 🧐

**Review by fehersanyi-bitrise — CHANGES_REQUESTED** [review]

**Review by fehersanyi-bitrise — DISMISSED** [review]
lgtm

File: README.md [hunk 2]
```diff
@@ -200,6 +200,9 @@ Once installed:
     add the Terminal and the Script Runner apps to `System Preferences - Security and Privacy - Accessability`
     also run the machine-setup/playbook.yml on the machine to enable access to system events
     you will need to run this from your dev machine, because the setting needs manual interaction, so the automation would time out.
+    
+* Add SSH key to authorized keys
+* Add `vagrant` user ad passwordless sudoer
```
Comment (bitrise-tom) [inline README.md:205]: s/ad/and

**Review by bitrise-tom — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (bitrisekristof) [reply README.md]: Fixed

**Review by bitrise-tom — APPROVED** [review]

**Review by Kanan-Bitrise — APPROVED** [review]

**Pull request merged**